src/string.rs

```diff
@@ -254,6 +255,22 @@ impl String {
     buffer: &mut [u8],
     nchars_ref: Option<&mut usize>,
     options: WriteOptions,
+  ) -> usize {
+    let buffer = unsafe {
```

Comment:
Can this method even be called in a safe way except for already safe buffers?

ie. Is this here only to not break the build and to enable a graceful transition into the `_uninit` API?

Reply:
Yes. Adding a new method will not break the build.

There is another way:

```rust
 pub fn write_utf8(
    &self,
    scope: &mut Isolate,
    buffer: OutBuf<'_>
    nchars_ref: Option<&mut usize>,
    options: WriteOptions,
  ) -> usize
```

where the `OutBuf` is something like [base64-simd::OutBuf](https://docs.rs/base64-simd/0.6.2/base64_simd/struct.OutBuf.html)